.github/workflows/cabal-ci.yml

```diff
@@ -1,19 +1,26 @@
 on:
   - push
   - pull_request
-name: cabal build with ghc 9.6
+name: cabal build
 jobs:
   runhaskell:
     name: cabal test
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        version:
+          - ghc: 9.6.3
+            cabal: 3.10.2.0
```

Comment:
```suggestion
```
We already have 9.6 covered by the stack matrix. Might as well not use it here. Could you remove `ghc96.cabal` files?